akamai-gtm.go

```diff
@@ -19,7 +19,7 @@ import (
 )
 
 var (
-	VERSION = "0.4.0"
+	VERSION = "0.5.0"
```

Comment:
Shouldn't we change the version (0.4.0 -> 0.5.0) during release? Or are we merging the M1 PR with release candidate already?

Reply:
we will merge it only on release date https://github.com/akamai/cli-gtm/pull/12#pullrequestreview-1413307940
We need to have all the changes here to create binaries for release